docs/OPENID4VCI-SATOSA-FRONTEND.md

```diff
@@ -292,6 +292,13 @@ Root section for credential management configuration.
 | config.credential_configurations.status_list.ttl          | Time-to-live for the status list (in minutes). Mandatory in `status_list` endpoint.                                                                                                                                                                                                                                                                                                                                                                        |
 | config.credential_configurations.credential_specification | A dictionary where each key corresponds to a credential type declared in `config.metadata.openid_credential_issuer.credential_configurations_supported` (metadata). Each value includes the `expiry_days` (validity in days) and a `template` for rendering the credential with placeholders in given format present in `config.metadata.openid_credential_issuer.credential_configurations_supported.<cred>.format`.  Mandatory in `credential` endpoint. |
 
+### Security 
+Root section for security configuration.
+
+| Key                           | Description                                                                                                       |
+|-------------------------------|-------------------------------------------------------------------------------------------------------------------|
+| config.security.dpop_required | This is the flag to enable the DPOD interoperability and validation, if missing it will be set to true by default |
```

Comment:
```suggestion
| config.security.dpop_required | This is the flag to enable the DPOP interoperability and validation, if missing it will be set to true by default. |
```